app/controllers/users.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('../logger');
+const userService = require('../services/users');
+const { registerBodyMapper } = require('../mappers/users');
+const errors = require('../errors');
+
+exports.registerUser = (req, res, next) => {
+  logger.info('Starting the user creation');
+  const user = registerBodyMapper(req.body);
+  return userService
+    .validateUser(user)
+    .then(responseUser => {
+      if (!responseUser) return Promise.reject(errors.databaseError('user already exists'));
+      return userService
+        .registerUser(user, req)
+        .then(response => {
+          logger.info(response.name);
+          logger.info('Success user creation');
```

Comment:
you can log one thing. For example: `logger.info('User with name: ${response.name} and email: ${response.email} was created')` 
PD: In the string is ` instead of ' :see_no_evil: